### DISCUSSION TITLE
Font rendering changed somehow

### DISCUSSION THREAD

**Discussion by BenKenobie31415** [polybar/polybar#3146] [Q&A] [answered]
I use FiraCode Nerd Font Mono with [this polybar config](https://pastebin.com/yACRfG4J) After updating via my package-manager, the font renders different now (kind of stretched?). I didn't change anything in the config in the meantime.
Previously:
![1723838724](https://github.com/user-attachments/assets/e614aeb3-51eb-402a-a6c1-acad4bea6c9d)
Now:
![1723838730](https://github.com/user-attachments/assets/8a72f75b-b1b5-4ffd-9e86-1c8b08ed253d)

I could't find anything on this and maybe I overlooked something but I couldn't solve the issue yet.

WM: bspwm

**Comment by patrick96** [comment]
Do you know between which versions of polybar you updated?

**Reply by BenKenobie31415** [reply]
I am now on 3.7.1-1, not 100% certain about the previous version but I update every other day so very likely to be the previous one.

**Reply by patrick96** [reply]
The 3.7.1 release has been out a while. So most likely something about your fonts changed. Check if polybar loads a different file now:

https://github.com/polybar/polybar/wiki/Fonts#finding-font-names

**Reply by BenKenobie31415** [reply]
I tried around a bit and it seems to have to do with the font scaling.
When I change the size around, sometimes the text looks stretched and sometimes doesn't.
But I couldn't figure out a size where it isn't too small or too big and still looks like before.
I'm quite confused, I looked at all different styles shown with fc-list but that didn't really do anything about my problem since this is across all styles and it is definitely the correct font that is being displayed.

**Reply by BenKenobie31415** [reply, marked as answer]
At this point I looked at this damn font for so long, I can't recall what it should look like, so I'll probably just stop and ignore it 🤷 
In case I find something new, I'll post but I doubt it will happen...
